### ISSUE TITLE
chapter 11 Momentum algorithm

### ISSUE THREAD

**Issue by nitml** [ageron/handson-ml#269]
I'm having trouble understanding momentum algorithm part in chapter 11: in book it was written that 
**m** ← β**m** + learning_rate * gradient_of_cost_function      ....equation (a)
**θ**<=**θ** - **m**       ... equation (b)

after this it was written if gradient is constant, the terminal velocity is equal to the gradient multiplied by the learning_rate * (1/(1-β)).
 
how can  we say this ? 
is terminal velocity mean's maximum **θ** ?Is this by taking β**m** of equation (a) to left and calculating "**m**"  and putting it equation (b)? if so then seeing just equation(b) we can calculate "**m**" equal to zero.

**Comment by ageron** [comment]
Hi @nitml ,

Thanks for your question.  Let's suppose that the gradient is constant and that the algorithm eventually reaches a terminal velocity. Let's call **m**\_terminal the momentum when this terminal velocity is reached. By definition, since it is a "terminal" velocity, it means that it does not change anymore, so in particular equation (a) should not change it. So we can write:

**m**\_terminal = β**m**\_terminal + learning_rate * gradient_of_cost_function

We can rewrite this as:

**m**\_terminal - β**m**\_terminal = learning_rate * gradient_of_cost_function

Therefore:

**m**\_terminal (1 - β) = learning_rate * gradient_of_cost_function

So:

**m**\_terminal = learning_rate * gradient_of_cost_function / (1 - β)

You can think of this as jumping off an airplane: at first you will accelerate quickly, then the wind will start to grow, and after a while the force of the wind pushing you up will be equal to the force of gravity pulling you down: at this point, you will no longer accelerate, you will have reached terminal velocity. The thinner the atmosphere, the faster the terminal velocity. In this analogy, the atmosphere's "thinness" is β: when it is close to 1, there is almost no resistance and the terminal velocity is huge. When it is close to 0, there is a lot of resistance and the terminal velocity is low.

Hope this helps,
Aurélien

**Comment by nitml** [comment]
Thanks for explaining and for your time
